README.md

```diff
@@ -41,7 +41,9 @@ The pre-flight check requires a configuration file so it can perform the appropr
     },
     "tags": {
         "simple": "true"
-    }
+    },
+    "no_sandbox": false,
```

Comment:
we don't need to specify this if the value is false. It will default to using the sandbox unless we override it.